packages/esm-patient-chart-app/src/clinical-views/components/clinical-views-summary.component.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react';
+import { useConfig } from '@openmrs/esm-framework';
+import { getEncounterTileColumns } from '../../encounter-tile/utils/encounter-tile-config-builder';
+import {
+  MemoizedEncounterTile,
+  type EncounterTileColumn,
+} from '../../encounter-tile/components/encounter-tile.component';
+
+interface OverviewListProps {
+  patientUuid: string;
+}
+
+interface TileDefinition {
+  title: string;
+  columns: Array<EncounterTileColumn>;
+}
+
+const ClinicalViewsSummary: React.FC<OverviewListProps> = ({ patientUuid }) => {
+  const config = useConfig();
+  const tilesDefinitions = config.tilesDefinitions;
+
+  const tilesData = useMemo(
+    () =>
+      tilesDefinitions?.map((tile: any) => ({
```

Comment:
This should be correctly typed.

```suggestion
      tilesDefinitions?.map((tile=> ({
```